scaling_performance/network_optimization.adoc

```diff
@@ -0,0 +1,142 @@
+[[scaling-performance-routing-network-optimization]]
+= Routing and Network Optimization
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+
+[[scaling-performance-scaling-router-haproxy]]
+== Scaling {product-title} HAProxy Router
+
+The {product-title}
+xref:../install_config/router/index.adoc#install-config-router-overview[router]
+is the ingress point for all external traffic destined for {product-title}
+services.
+
+On an Amazon EC2 m4.xlarge instance, a single HAProxy router is able to handle
+between 7000-32000 HTTP keep-alive requests depending on encryption, page size,
+and the number of connections used. For example, when using TLS
+xref:../architecture/core_concepts/routes.adoc#edge-termination[edge] or
+xref:../architecture/core_concepts/routes.adoc#re-encryption-termination[re-encryption]
+terminations with large page sizes and a high numbers of connections, expect to
+see results in the lower range. With HTTP keep-alive, a single HAProxy router is
+capable of saturating 1 Gbit NIC at page sizes as small as 8 kB.
+
+The table below shows HTTP keep-alive performance on such a public cloud
+instance with a single HAProxy and 100 routes:
+
+[cols="2,3,3"]
+|===
+|Encryption |Page size |HTTP(s) requests per second
+|none |1kB |15435
+|none |4kB |11947
+|edge |1kB |7467
+|edge |4kB |7678
+|passthrough |1kB |25789
+|passthrough |4kB |17876
+|re-encrypt |1kB |7611
+|re-encrypt |4kB |7395
+
+|===
+
+However, on Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50 GHz, you can expect roughly
```

Comment:
Please change as per reviewed: https://docs.google.com/document/d/1A_Lzym4s7Jg-f2O11VGAijr4IeNPrpnq5QkKlxGGKiI/edit

When running on bare metal with modern processors, you can expect roughly